applications/nrf_desktop/src/modules/hid_state.c

```diff
@@ -1047,6 +1103,14 @@ static void disconnect(const void *subscriber_id, uint8_t report_id)
 		return;
 	}
 
+	bool update_kbd_leds = false;
+
+	if (IS_ENABLED(CONFIG_DESKTOP_HID_REPORT_KEYBOARD_SUPPORT) &&
```

Comment:
can we at this level have it a bit more generic? something like for report_send - check comment on 1437

Reply:
I wonder if we should introduce more generic approach here. Handling HID output reports "subscriptions" seems not generic.